src/App/App.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { MemoryRouter } from 'react-router-dom';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import App from './App';
+import { getCocktailDetails, getRandomCocktail } from "../apiCalls";
+import { act } from 'react-dom/test-utils';
+import { cocktail } from '../test-data';
+jest.mock('../apiCalls');
+
+describe('App', () => {
+	getCocktailDetails.mockResolvedValue(() => {
```

Comment:
We need to figure out the error we're both getting with mockResolvedValue in TS

Reply:
Agreed